src/smt_db.rs

```diff
@@ -3,22 +3,21 @@ use std::collections::HashMap;
 use crate::consts;
 use crate::smt;
 
-pub struct SMTDB<'a> {
-    db: Box<&'a rocksdb::DB>,
+pub struct SmtDB<'a> {
```

Comment:
We usually use capital for abbreviated terms (ex: HTTP). Do we need to change this?

Reply:
Yes, the linter `clippy` suggested that changes based on the following error https://rust-lang.github.io/rust-clippy/master/index.html#upper_case_acronyms.